tssc/step_implementers/package/maven.py

```diff
@@ -69,7 +69,7 @@ def _run_step(self, runtime_step_config):
         if not os.path.exists(pom_file):
             raise ValueError('Given pom file does not exist: ' + pom_file)
 
-        process_args = ["mvn", "clean", "install"]
+        process_args = 'mvn clean install'
```

Comment:
Just curious why this was changed from a list to a string.

Reply:
It's strange, but for some reason using `shell=True` fails with a list, but works with a string.